main.go

```diff
@@ -45,11 +45,15 @@ var (
 	argElasticURL = ""
 	argUsername   = ""
 	argPassword   = ""
+	argRepoAccessKey = ""
+	argRepoSecretKey = ""
+	argRepoRegion = "eu-west-1"
 	argUseSSL     = true
+	argUseRepoAuth = false
```

Comment:
I think we could remove this key and just use the existence of `argRepoAcessKey` & `argRepoSecretKey` to determine if we want to do an authenticated request.

Reply:
The UseRepoAuth key has been removed and the Authenticated request now depends on existence of (both) Access and Secret keys.